Dockerfile

```diff
@@ -1,20 +1,18 @@
 FROM java:openjdk-8
-ENV MAXWELL_VERSION 1.10.7
-ENV KAFKA_VERSION 0.10.1.0
+ENV MAXWELL_VERSION=1.10.7 KAFKA_VERSION=0.10.1.0
 
-RUN apt-get update && apt-get -y upgrade
-
-RUN apt-get install build-essential ruby -y
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && apt-get -y install build-essential maven
 
 COPY . /workspace
-WORKDIR /workspace
-
-RUN KAFKA_VERSION=$KAFKA_VERSION make package MAXWELL_VERSION=$MAXWELL_VERSION
-
-WORKDIR /workspace/target/maxwell-$MAXWELL_VERSION/
 
-RUN mkdir /app \
-  && mv ./* /app/
+RUN cd /workspace \
+    && KAFKA_VERSION=$KAFKA_VERSION make package MAXWELL_VERSION=$MAXWELL_VERSION \
+    && mkdir /app \
+    && mv /workspace/target/maxwell-$MAXWELL_VERSION/maxwell-$MAXWELL_VERSION/* /app/ \
+    && apt-get clean \
```

Comment:
`apt-get clean` here doesn't reduce size, since the files have been generated (and cached) in the previous layer.